### ISSUE TITLE
Better Quick Start Guide

### ISSUE THREAD

**Issue by arosen93** [materialsproject/atomate2#405]
I think we should strive to have a better quick start guide in the docs. Not using VASP. Something that can be run on the local machine and get you up and running in as quickly as possible just as a proof-of-concept.

**Comment by JaGeo** [comment]
I agree that an example with the forcefields would be cool. However, an optimization only might not yet be a workflow.

**Comment by arosen93** [comment]
Many people will want to run a single job, so a one-step workflow is okay for a quick start! Want to make it a workflow? Do an optimization with M3GNet and take the output into CHGNet to see how much they differ. Now you have a workflow!

**Comment by arosen93** [comment]
Or use your upcoming phonon workflow ;)

**Comment by JaGeo** [comment]
This should probably motivate us to be faster with the phonon workflow  or others to provide other workflows including the forcefields. ;)

**Comment by mjwen** [comment]
Related to lowing the entry barrier: I think it should also be helpful to use an on-disk DB like MontyDB in maggma in the Installation section of the doc. Figuring out how to set up MongoDB, e.g., on Atlas is nontrivial for many.  

Of course, we can move the MongoDB stuff to another tutorial named Production Environment and such.
